src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -5,9 +5,14 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Globalization;
+#if !NETSTANDARD1_3
```

Comment:
That's a bummer.

Reply:
Yes, the ClaimsPrincipal class is not available for NetStandard1.3.

Reply:
It is available in netstandard2.0+, if we move to that.